cmd/serve.go

```diff
@@ -106,7 +108,8 @@ func serve() {
 	}
 
 	go func() {
-		if err := srv.ListenAndServe(); err != nil {
+		logger.Security().SystemStartup()
+		if err := srv.ListenAndServe(); err != nil && !errors.Is(err, http.ErrServerClosed) {
```

Comment:
is the one below caught, aka if the goroutine exits with a fatal logging, do we crash and fulfill the auditing?

Reply:
No I think that if the server exits we don't catch it. This change is there so that we can properly catch signals, before if I killed the server (ctrl+c) the logs were not showing up.

Reply:
If we want to fix that in this PR we could add a defer func calling the recover()
```
defer func() {
        if r := recover(); r != nil {
            fmt.Println("Recovered in f", r)
        }
}()
```